maestro-cli/src/main/java/maestro/cli/command/CloudCommand.kt

```diff
@@ -139,9 +140,15 @@ class CloudCommand : Callable<Int> {
     @Option(hidden = true, names = ["--fail-on-cancellation"], description = ["Fail the command if the upload is marked as cancelled"])
     private var failOnCancellation: Boolean = false
 
+    @Option(hidden = true, names = ["--fail-on-timeout"], description = ["Fail the command if the upload times outs"])
+    private var failOnTimeout: Boolean = true
+
     @Option(hidden = true, names = ["--disable-notifications"], description = ["Do not send the notifications configured in config.yaml"])
     private var disableNotifications = false
 
+    @Option(hidden = true, names = ["--result-wait-timeout"], description = ["Minutes to wait until all flows complete"])
```

Comment:
```suggestion
    @Option(hidden = true, names = ["--timeout"], description = ["Minutes to wait until all flows complete"])
```

What about just `--timeout` as arg name?

Reply:
I thought it would be too generic but then again, there aren't other cases for timeout for this command.